recipes/newrelic/infrastructure/suse.yml

```diff
@@ -42,7 +42,7 @@ install:
           fi
           sudo touch /etc/newrelic-infra.yml;
         - |
-          echo -e "license_key: {{.NEW_RELIC_LICENSE_KEY}}" | sudo tee -a /etc/newrelic-infra.yml > /dev/null
+          grep -q '^license_key' /etc/newrelic-infra.yml && sudo sed -i 's/^license_key.*/license_key: {{.NEW_RELIC_LICENSE_KEY}}/' /etc/newrelic-infra.yml || echo 'license_key: {{.NEW_RELIC_LICENSE_KEY}}' | sudo tee -a /etc/newrelic-infra.yml > /dev/null
```

Comment:
The order things are happening here is confusing to me. We removed/recreated the `newrelic-infra.yml` on lines 40-43, then are grepping for a license key in the file we just created? I thought this was removed at some point in the past...

Reply:
I've copy/pasted from the other infra recipes.